src/tito/tagger/main.py

```diff
@@ -243,10 +239,21 @@ def _make_changelog(self):
 
                 write(fd, header)
 
-                for cmd_out in output.split("\n"):
-                    write(fd, "- ")
-                    write(fd, "\n  ".join(textwrap.wrap(cmd_out, 77)))
+                # don't die if this is a new package with no history
+                if self._changelog_msg is not None:
+                    write(fd, self._changelog_msg)
```

Comment:
Is there automatic support for multi-args or an example you know of off hand? I think adding '--changelog' as a multi-option will be a fine approach as for my usage I can still send a block with '--changelog'.